### PR TITLE
Hook up BCNM Timer Packet

### PR DESCRIPTION
**Pull request by zach2good**
**_I affirm:_**
- [x] that I agree to Project Topaz's [Limited Contributor License Agreement](http://project-topaz.com/blob/release/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

File: scripts/globals/bcnm.lua [hunk 1]
```diff
@@ -973,7 +973,7 @@ function EventTriggerBCNM(player, npc)
     elseif not player:hasStatusEffect(tpz.effect.BATTLEFIELD) then
         local mask = findBattlefields(player, npc, 0)
 
-        -- mask = 268435455 -- uncomment to open menu with all possible battlefields
+        mask = 268435455 -- uncomment to open menu with all possible battlefields
```
Comment (rude-jerk) [inline scripts/globals/bcnm.lua]: Whoops!

Comment (zach2good) [reply scripts/globals/bcnm.lua]: image

File: src/map/battlefield.cpp [hunk 2]
```diff
@@ -288,6 +289,7 @@ bool CBattlefield::InsertEntity(CBaseEntity* PEntity, bool enter, BATTLEFIELDMOB
                 ApplyLevelRestrictions(PChar);
                 m_EnteredPlayers.emplace(PEntity->id);
                 luautils::OnBattlefieldEnter(PChar, this);
+                charutils::sendTimerPacket(PChar, m_TimeLimit);
```
Comment (zach2good) [inline src/map/battlefield.cpp]: This should be SendTimerPacket

**Comment by zach2good** [comment]
This has existed (and been used in Full Speed Ahead) since the summer, probably fine 🤷

**Pull request merged**